src/main/java/org/diatliuk/bookstore/dto/user/UserLoginRequestDto.java

```diff
@@ -0,0 +1,17 @@
+package org.diatliuk.bookstore.dto.user;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Data;
+import org.hibernate.validator.constraints.Length;
+
+@Data
+public class UserLoginRequestDto {
+    @NotBlank
+    @Schema(example = "new@example.com")
+    private String email;
```

Comment:
To my mind, using a custom annotation with regex pattern covering more cases for different emails patterns. Also think about if the field is empty. Maybe add more cases to check data validation.

Reply:
Yes, you're right. My bad, I forgot to specify